configure.ac

```
@@ -2986,8 +2986,12 @@ AC_HEADER_DIRENT
 AC_HEADER_MAJOR
 
 # for faulthandler
-AC_CHECK_HEADERS([execinfo.h link.h dlfcn.h],
-                 [AC_CHECK_FUNCS(backtrace backtrace_symbols dladdr1)])
+AC_CHECK_HEADERS([execinfo.h link.h dlfcn.h], [
+  AC_CHECK_FUNCS([backtrace backtrace_symbols dladdr1], [
```

backtrace_symbols() can be removed, it's no longer used:

```suggestion
  AC_CHECK_FUNCS([backtrace dladdr1], [
```

---

I was planning on doing this in a followup, but it's fine to do it here as well.